static/src/stylesheets/module/site-messages/_fiv-banner.scss

```diff
@@ -478,12 +478,7 @@ $wide-circle-height: 162px;
         color: $brightness-100;
         border:  $brand-main 1px solid;
     }
-
-    .engagement-banner__button.engagement-banner__button__learn-more {
```

Comment:
there are 2 more occurrences in this file that should probably be removed

Reply:
of `engagement-banner__button__learn-more`

Reply:
✅